common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -45,6 +45,10 @@ class @Problem
     @saveNotification = @$('.notification-save')
     @saveButtonLabel = @$('.action .save .save-label')
     @saveButton.click @save
+    @gentleAlertNotification = @$('.notification-gentle-alert')
+
+    # Hide the Gentle Alert notification until needed
+    @gentleAlertNotification.hide()
```

Comment:
I'd recommend initially rendering with it hidden to avoid any potential visual glitches during slow rendering.